src/js/Content/Modules/SteamFacade.js

```diff
@@ -86,16 +86,13 @@ class SteamFacade {
 
     // tooltips
 
-    static bindTooltips(selector, rgOptions) {
-        BindTooltips(selector, rgOptions);
-    }
-
-    static setupTooltips(className = "community_tooltip") {
-        return SetupTooltips({"tooltipCSSClass": className});
-    }
-
-    static vTooltip(selector, method) {
-        $J(selector).v_tooltip(method);
+    static vTooltip(selector, isStore = false, isHtml = false) {
+        $J(selector).v_tooltip({
+            "tooltipClass": isStore ? "store_tooltip" : "community_tooltip",
```

Comment:
Couldn't we automatically deduce this from the host name?

Reply:
It's likely fine, but still tricky. What is something changes in the future?

Reply:
You mean the domain?
Then our scripts aren't even running

Reply:
I think just adding a comment that you're assuming the class name and domain are related should be fine enough.

Reply:
The day this changes is probably the day Steam stops using jQuery...